scripts/entrypoint.sh

```diff
@@ -2,18 +2,33 @@
 
 # Default values:
 : "${INPUT_MAX_LAYERS:=-1}"
+: "${INPUT_SHOW_CURRENT_SIZE:=false}"
```

Comment:
This needs two steps to be properly initialized: see https://github.com/wemake-services/wemake-python-styleguide/blob/fa815ebfb3fd1457712812597de4c1320f83e1b9/scripts/entrypoint.sh#L3-L16